jsonnet/kube-prometheus/jsonnetfile.json

```diff
@@ -17,7 +17,7 @@
           "subdir": "contrib/mixin"
         }
       },
-      "version": "master"
+      "version": "562d645ac923388ff5b8d270b0536764d34b0e0f"
```

Comment:
any reason we pin this?

Reply:
ah, nevermind, we're cutting and pinning things here :facepalm:

Reply:
We had issues in the past with moving targets in release branches. See all PRs with etcd mixin moving to a different directory. Pinning in release branches should prevent such issues in the future.